modules/vectra.py

```diff
@@ -29,6 +30,15 @@ def api_validator(self, *args, **kwargs):
     return api_validator
 
 
+def validate_python_v3(func):
```

Comment:
Note, this is fine.  

A cleaner solution is to add a dependency on nine, https://pypi.org/project/nine/, then you could use IS_PYTHON_2.